devlab/tests/migrate.yaml

```diff
@@ -52,18 +52,15 @@ process:
       - check_instances: ['rename_info_iter']
       - get_next_instance: True
       - trans_one_inst:
+          - act_detaching_src_volume: False
+          # after migration volume will be attached on src and dst at same time
```

Comment:
Also, please rename the task to `detach_volumes_on_source`, it's much more readable.